### DISCUSSION TITLE
Integrating Milvus with GKE Anthos Service Mesh

### DISCUSSION THREAD

**Discussion by iameminmammadov** [milvus-io/milvus#29065] [Q&A and General discussion]
Hey guys,

Trying to integrate Milvus with ASM. It reaches https: // DNS with 443 but fails to get response from gRPC. I get 404 from POST in K8s logs. Any tips? I double checked DestionationRule and they seem fine.

**Comment by wyfeng001** [comment]
Dup of #29096.

Close #29065
